tests/managed-service/test_ms_markers.py

```diff
@@ -0,0 +1,96 @@
+import logging
+import pytest
+
+from ocs_ci.framework.testlib import (
+    tier1,
+    ManageTest,
+    ignore_leftovers,
+    managed_service_required,
+    skipif_ms_consumer,
+    skipif_ms_provider,
+    runs_on_provider,
+)
+from ocs_ci.ocs.cluster import (
+    is_ms_consumer_cluster,
+    is_ms_provider_cluster,
+    ceph_health_check,
+)
+from ocs_ci.ocs.managedservice import (
+    check_default_cluster_context_index_equal_to_current_index,
+)
+
+logger = logging.getLogger(__name__)
+
+
+@tier1
+@managed_service_required
+@ignore_leftovers
+class TestManagedServiceMarkers(ManageTest):
+    """
+    Test that the managed service markers work as expected
+    """
+
+    @pytest.fixture(autouse=True)
+    def teardown(self):
```

Comment:
Ceph health should be checked automatically after all tests. If this is missing in case of multi cluster then this needs to be handled in a more general way that works for all tests. Please create an issue if it is missing.

Reply:
Yes, I can remove it.